non-technical/self-introduction.md

```diff
@@ -5,6 +5,23 @@ You can rephrase the question like this:
 
 "Tell me about your journey into tech. How did you get interested in coding, and why was web development a good fit for you? How is that applicable to our _____ role or company goals?"
 
+### The Elevator Pitch
+
+The Elevator Pitch is an indispensable tool for you as you move forward in your career. An Elevator Pitch is just that -- you pitch yourself to an executive that you want to impress and only have a short elevator ride to do so. Whether you're at a job fair with hundreds of other candidates and you have limited time or you are simply explaining who you are to a potential connection or client, it is important to be able to clearly and accurately describe your knowledge and skillset quickly and succinctly. Here are some tips to develop a good Elevator Pitch:
+
+- Sell yourself
+  - The whole point of this is to get you a job or make a connection that benefits your career.
+  - Tell them who you are, who you work for (or school and major), and what you do.
+- KISS (Keep It Simple, Stupid)
+  - Tell them some highlights from your favorite / most impressive projects.
+    - Do not delve into the depths of how you reverse engineered a game and decrypted a packet to predict when to use your DKP on a drop. Tell them the executive summary: "I reverse engineered X game by decrypting Y packet to predict Z." If this catches their interest, they *will* ask further questions on their own.
```

Comment:
Is this line meant to be have a deeper indentation or should it be on the same level as "Tell them some highlights..."?

Reply:
I meant it to be a deeper indentation, yes. Feel free to change! Just trying to help some others out from personal experience, not picky how it gets there! :)